Editor/AGS.Editor/Panes/ViewPreview.cs

```diff
@@ -113,20 +113,17 @@ private void previewPanel_Paint(object sender, PaintEventArgs e)
                     spriteWidth *= 2;

```

Comment:
A sprite size is doubled right above your new code under same condition, it should be possible replace hardcoded x2 with GUIScaleFactor and perhaps use same variables when drawing too, instead of having (n * scale) everywhere?